packages/select/src/MenuButton.tsx

```diff
@@ -39,28 +40,32 @@ const menuButtonStyleOverrides = css`
   }
 `;
 
+// We don't have 12px or 10px in our spacing vars
+const _12 = spacing[2] + spacing[1];
+const _10 = spacing[2] + spacing[1] / 2;
```

Comment:
Not a big fan of these variants - I think I'd rather see a more descriptive name like `const rightPadding = 10` and see that reused throughout.